api/src/task/certificates/setup_cert_user_task.py

```diff
@@ -0,0 +1,124 @@
+import datetime
+import logging
+import uuid
+
+import click
+from sqlalchemy import select
+
+import src.adapters.db.flask_db as flask_db
+from src.adapters import db
+from src.constants.lookup_constants import UserType
+from src.db.models import staging
+from src.db.models.user_models import (
+    Agency,
+    AgencyUser,
+    AgencyUserRole,
+    LegacyCertificate,
+    Role,
+    User,
+)
+from src.task.ecs_background_task import ecs_background_task
+from src.task.task import Task
+from src.task.task_blueprint import task_blueprint
+
+logger = logging.getLogger(__name__)
+
+
+@task_blueprint.cli.command("setup-cert-user", help="Setup the LegacyCertificate and User")
+@click.option("--cert-id", "-t", help="currentcertid on Tcertificate")
```

Comment:
Probably a bad time to ask this question, but is this cert ID something secretive that we should avoid logging? If so, we might need to do things a bit different as even if you didn't explicitly log it, the command gets logged.